LICENSE.txt

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2012-2021 Authy Inc
+Copyright 2021 Jay Wolff
```

Comment:
Copyright should be SPRX, Inc since it was done under employment but you can certainly list yourself as author/contributor. Copyright year should also be updated and we should host this under Injuryscripts repository (fine if you want a copy under your own account too since we don't have any reason for it to not remain MIT at this time).